### PR TITLE
Fix column vector

### PR DESCRIPTION
**Pull request by yasirroni**

### CODE REVIEW

**Comment by yasirroni** [comment]
Done @blink1073

**Comment by yasirroni** [comment]
Fix https://github.com/blink1073/oct2py/issues/221

File: oct2py/io.py [hunk 1]
```diff
@@ -49,7 +49,11 @@ def read_file(path, session=None):
 def write_file(obj, path, oned_as='row', convert_to_float=True):
     """Save a Python object to an Octave file on the given path.
     """
+    # print(obj)
```
Comment (blink1073) [inline oct2py/io.py:52]: Can you please remove the commented out print statements here and in the test file?

**Review by blink1073 — COMMENTED** [review]
Thanks!  GitHub Actions is having an outage, and I opened https://github.com/Calysto/metakernel/pull/244 to fix the test warnings we're seeing.  I'll make a release of metakernel and kick this to make sure tests pass once GitHub Actions is running a gain.

**Comment by blink1073** [comment]
Kicking again

File: oct2py/io.py (hunk 1 above)
Comment (yasirroni) [reply oct2py/io.py]: Sure!

**Comment by yasirroni** [comment]
If this is merged, I hope you make it as release for new version since my package depends on it.

**Comment by blink1073** [comment]
> If this is merged, I hope you make it as release for new version since my package depends on it.

Yep, I'll cut a release tomorrow.  Thanks again!

**Pull request merged**